commands/station.js

```diff
@@ -22,6 +23,8 @@ const panic = msg => {
 }
 
 export const station = async ({ json, experimental }) => {
+  const STATION_ID = (await getStationId()).publicKey
```

Comment:
@bajtos do you have thoughts on whether UPPER_CASE should be reserved for environment variables, or is it for constants in general?

Reply:
I don't have a strong opinion. 

We are using UPPER_CASE for many fields in the options parameters passed around, so I thought it would be more consistent to use `STATION_ID` here. (Compare this name to `FIL_WALLET_ADDRESS` referenced in this file.)

IMO, variable names should describe what value is stored in them. How we initialise the name is a detail that can vary, therefore it should not influence the name. For example, in tests, we often set FIL_WALLET_ADDRESS to a hard-coded value.

Personally, I would do this:

```js
const {
  FIL_WALLET_ADDRESS: walletAddress,
  // ...
} = process.env

// then use `walletAddress` everywhere else
```

Reply:
Ok! Great that you value consistency more!

> Personally, I would do this:

Thinking about it some, I would also prefer this. Let's address it in the future if we feel strong enough about it